code/modules/surgery/surgery_step.dm

```diff
@@ -1,3 +1,40 @@
+//Overrides the surgery step to require anasthetics for a smooth surgery
+/datum/surgery_step/initiate(mob/user, mob/living/target, target_zone, obj/item/tool, datum/surgery/surgery, try_to_fail = FALSE)
```

Comment:
to nie powinno być po definicji /datum/surgery_step?

Reply:
Całość w oryginale była dokładnie tu

Reply:
na samym końcu jest taka sama funkcja, coś mi tu śmierdzi
![image](https://user-images.githubusercontent.com/50628162/102506008-f5e0ab00-4082-11eb-88d8-19b2a02821aa.png)